pkg/bpf/map_linux.go

```diff
@@ -155,6 +159,13 @@ func (m *Map) Flags() uint32 {
 	return 0
 }
 
+func (m *Map) updateMetrics() {
```

Comment:
This and the call to `metrics.UpdateMapCapacity()` don't perform any update AFAIU, looks like something is missing to perform an actual operation?

Reply:
Woops, was making some last minute refactors 😅 ... Fixed.